sdk/log/record_test.go

```diff
@@ -635,3 +640,12 @@ func TestTruncate(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkSetAddAttributes(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		kv := log.String("key", "value")
+		var r Record
+		r.SetAttributes(kv)
+		r.AddAttributes(kv)
+	}
```

Comment:
The `kv` and `r` declarations should not be included in the benchmark, they are unrelated to the tested methods. Also, you are showing memory benchmarks, but this does not output any memory benchmarks.

```suggestion
	kv := log.String("key", "value")
	records := make([]Record, b.N)

	b.ReportAllocs()
	b.ResetTimer()
	for i := 0; i < b.N; i++ {
		records[i].SetAttributes(kv)
		records[i].AddAttributes(kv)
	}
```